app/views/thredded/posts_common/actions/_quote.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= link_to t('thredded.posts.quote_btn'), url_for(post.quote_url_params),
+<%= link_to t('thredded.posts.quote_btn'), post.quote_url,
```

Comment:
@glebm I'm just re-reviewing this one (after a looong while! It would be good to just close it or) and I realise that this is in fact the only case where we use a bare `url_for` (rather than Thredded::UrlsHelper.url_for) and it might be really hard work to find the case where this is a problem -- might be better to avoid using url_for just defensively?

but if you disagree, we should close this anyway.

Reply:
Sure, though if we extract it into a method let's name it something other than `quote_url` because there is already `quote_path` which is something else.